src/Symfony/Bridge/Twig/Command/DebugCommand.php

```diff
@@ -29,15 +29,27 @@ class DebugCommand extends Command
     private $twig;
 
     /**
-     * {@inheritdoc}
+     * @param Environment $twig
      */
     public function __construct($name = 'debug:twig')
```

Comment:
Not sure.. did that initially. But given we dont have to change it, this would be safest. So for BC really :)